### PR TITLE
IBX-6649: Spell checking support described

### PR DESCRIPTION
**Pull request by julitafalcondusza**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | (https://github.com/ibexa/admin-ui/pull/923)
| Versions      | master

Spell checking support described.

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date

### CODE REVIEW

File: docs/search/search_for_content.md [hunk 1]
```diff
@@ -26,6 +26,11 @@ By selecting an option from a drop-down menu in the top right corner, you can so
 
 ![Sorting menu](img/sorting_menu.png)
 
+The search engine not only displays suggestions, but also supports spell checking.
+If there is a typo in your search phrase, the search engine suggests a related, accurate term.
```
Comment (adriendupuis) [inline docs/search/search_for_content.md:30]: It's not only on typo it seems.


[basic_search.png](https://ez-systems-developer-documentation--290.com.readthedocs.build/projects/userguide/en/290/search/img/basic_search.png) also 🪄 "put a spell on user"

I don't know exactly why it was triggered in my search example but it worth a check.
(on low result count? by comparing terms' result counts?)

Comment (mnocon) [inline docs/search/search_for_content.md:30]:
```suggestion
If there is a typo in your search phrase, the search engine suggests a related, more accurate term.
```

Comment (julitafalcondusza) [reply docs/search/search_for_content.md:30]: If I understand you correctly - it also shows suggestions. Yes, it does, and we have it described: https://doc.ibexa.co/projects/userguide/en/latest/search/search_for_content/.

Comment (julitafalcondusza) [reply docs/search/search_for_content.md]: _"If you misspell a search phrase..."_ - so it cannot be correct/accurate. Search engine will then show accurate term, so I'd keep like it is.

**Review by adriendupuis — APPROVED** [review]

File: docs/search/search_for_content.md (hunk 1 above)
Comment (mnocon) [reply docs/search/search_for_content.md]: Ok, I have to admit I misunderstood the sentence and thought you're talking about the results and their accuracy (not the search suggestion). Ok for me!

**Review by mnocon — APPROVED** [review]

**Pull request merged**